controllers/mattermost/mattermost/mattermost_test.go

```diff
@@ -389,6 +391,132 @@ func TestCheckMattermost(t *testing.T) {
 	})
 }
 
+func TestCheckMattermostAWSIngress(t *testing.T) {
```

Comment:
This general tests are good, but I would are more checks, for example:
- Enabling/Disabling the AWS Load Balancer
- Switching from AWS Load Balancer to an Ingress (and vice-versa), all resources regarding both should be taken care of

Reply:
Il work on better tests in the next commit.

Reply:
Hey @Hunter-Thompson , thanks for the fast response! Just to clarify: The first ones are good to have, the others are nice to have! Just wanted to be sure that we disable or remove any spec everything is properly taken care of.